### ISSUE TITLE
unmaintained repo?

### ISSUE THREAD

**Issue by yogo1212** [jlund/ansible-go#37]
@jlund still active?

**Comment by xiegeo** [comment]
Still works well for me, I update the go versions manually as that is the best for reproducibility.

**Comment by yogo1212** [comment]
the code in this repo doesn't check the authenticity of downloaded binary and downloads an old version by default.
the repo owner doesn't respond..

my PR aims to fix the first issues along with some overall improvements.
but we can't get these things reviewed and merged without the help of @jlund.

**Comment by yogo1212** [comment]
i think it's safe to say that the repo owner has abandoned this project.
there's an open PR that addresses unnecessary downloading, missing signature checks, and fixing the go version through a variable.
https://github.com/jlund/ansible-go/pull/34 (merged in https://github.com/yogo1212/ansible-go)
that should obsolete updates to the hard-coded go version and redundant efforts to fix other issues - if it was visible to users.

where do we want to go?
i use this code and would be interested to have it move to a place where it can be developed more actively/collaboratively.

**Comment by xiegeo** [comment]
Good idea, I'd be happy to follow your fork, espsially if the role can be used from Anible Galaxy.

**Comment by yogo1212** [comment]
@xiegeo https://galaxy.ansible.com/yogo1212/ansible_go
let me know, if something is off ;-)
